engine/classes/Elgg/Gatekeeper.php

```diff
@@ -156,23 +156,11 @@ public function assertAccessibleEntity(ElggEntity $entity, ElggUser $user = null
 			if (!$this->session->getIgnoreAccess() && !$this->access->hasAccessToEntity($entity, $user)) {
 				// user is logged in but still does not have access to it
 				$msg = $this->translator->translate('limited_access');
-				$exception = new EntityNotFoundException($msg);
-				$exception->setParams([
```

Comment:
huh why did you remove this? you added this recently am i right?

Reply:
Eh, weird. I will have to redo the PR